src/map/script.cpp

```diff
@@ -8437,6 +8437,43 @@ BUILDIN_FUNC(delitem2)
 	return SCRIPT_CMD_FAILURE;
 }
 
+/// Deletes items from the target/attached player at given index.
+/// delitemidx <index>{,<amount>{,<char id>}};
+BUILDIN_FUNC(delitemidx) {
+	struct map_session_data* sd;
+	int amount = 0;
```

Comment:
```suggestion
```
Move this down to right before the if statement line where it's assigned and make it:
```cpp
int amount;
```